### ISSUE TITLE
Switch from `Nullable` generic to `<type> | null`

### ISSUE THREAD

**Issue by karlhorky** [ts-safeql/safeql#66]
**Is your feature request related to a problem? Please describe.**

It was confusing to me that this does not work:

```ts
type Animal = {
  id: number;
  name: string | null;
}

// error: type of name `string | null` does not match `Nullable<string>`
await sql<Animal[]>`
  SELECT * FROM animals
`
```

With this schema:

```sql
CREATE TABLE animals (
  id integer PRIMARY KEY GENERATED BY DEFAULT AS IDENTITY,
  name varchar(40)
)
```

**Describe the solution you'd like**

Support the `x | null` union type when using a nullable database field

**Describe alternatives you've considered**

Apparently as shown [in the examples](https://github.com/ts-safeql/safeql/blob/e14955ddbef2dea973f1550f96246d4aae063108/demos/big-project/src/original.ts#L3-L9), it shows that users can define their own `Nullable` generic type to achieve this:

```ts
type Nullable<T> = T | null;

type Animal = {
  id: number;
  name: Nullable<string>;
}

// no error
await sql<Animal[]>`
  SELECT * FROM animals
`
```

But this seems less natural and discoverable than allowing `| null` unions

**Additional context**

--

**Comment by Newbie012** [comment]
fixed in [0.0.15](https://github.com/ts-safeql/safeql/releases/tag/%40ts-safeql%2Feslint-plugin%400.0.15)

**Comment by karlhorky** [comment]
Amazing, works great! Upgraded here: https://github.com/upleveled/next-js-example-fall-2022/commit/ac7950e837174108236e77cebc3ac3fe8190f835
